### ISSUE TITLE
Issue when saving clefs

### ISSUE THREAD

**Issue by ATranimal** [DDMAL/Neon.js#139]
Continuation of #114 
There are two issues with saving clefs currently:

1. If you try to save a clef on the highest numbered system, you get a "Uncaught TypeError: Cannot read property 'id' of null", because the code tries to get the next system, which is null when inserting on the highest system.

2. If you try to save a clef anywhere else, you get "Uncaught TypeError: system.getActinglefByEle is not a function". Will look further into this

**Comment by ATranimal** [comment]
ignore 2. just realized theres a C messing in Clef

**Comment by ATranimal** [comment]
Fixed in 5d2e3091194fa19f970337fdff906d21050f8e20

**Comment by ATranimal** [comment]
New issue with saving clefs 

```
Traceback (most recent call last):
  File "/Users/andrewtran/Documents/Neon.js/app_env/lib/python2.7/site-packages/tornado/web.py", line 1509, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/Users/andrewtran/Documents/Neon.js/neonsrv/tornadoapi.py", line 297, in post
    result = md.insert_clef(line, shape, pitchInfo, before_id, ulx, uly, lrx, lry)
  File "/Users/andrewtran/Documents/Neon.js/neonsrv/modifymei.py", line 591, in insert_clef
    before = self.mei.getElementById(before_id)

ArgumentError: Python argument types in
    MeiDocument.getElementById(MeiDocument, unicode)

did not match C++ signature:
    getElementById(mei::MeiDocument {lvalue}, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >)
```

**Comment by ATranimal** [comment]
Fixed in 6071506a2cbffe5097f049b01cb0d2301d1341e7
